cmd/configschema/go.mod

```diff
@@ -722,6 +723,8 @@ replace github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostm
 
 replace github.com/open-telemetry/opentelemetry-collector-contrib/receiver/influxdbreceiver => ../../receiver/influxdbreceiver
 
+replace github.com/open-telemetry/opentelemetry-collector-contrib/receiver/iisreceiver => ../../receiver/iisreceiver
```

Comment:
Please add the component to the `cmd/configschema` tests as well.

Reply:
not seeing what test needs to be updated

Reply:
You're right. This has been abstracted away from this module. The `go.mod` change is all you need.